openlibrary/plugins/openlibrary/home.py

```diff
@@ -86,8 +84,8 @@ def get_ia_carousel_books(query=None, subject=None, work_id=None, sorts=None,
     if 'env' not in web.ctx:
         delegate.fakeload()
 
-    elif query in CAROUSELS_PRESETS:
-        query = CAROUSELS_PRESETS[query]
+    elif query in PRESET_QUERIES:
+        query = PRESET_QUERIES[query]
 
     limit = limit or lending.DEFAULT_IA_RESULTS
     books = lending.get_available(limit=limit, subject=subject, work_id=work_id,
```

Comment:
Wait, wasn't the point of this PR to remove this method?

Reply:
@cdrini no, it introduces the module which will deprecate `lending.get_available`.

The code review will quickly become massive if the entire thing were included in this PR.